test/e2e/lunchApp.js

```diff
@@ -0,0 +1,29 @@
+var conf = require('../../nightwatch.conf.js');
+const email = 'gbartrum2f@devbridge.com';
+const password = 'AyBrDdHAsc';
```

Comment:
uploading credentials to public repository is a security issue.

Reply:
login credentials should probably be stored somewhere else then in test case. login.js? credentials.js?